docs/eventing/samples/kafka/channel/README.md

```diff
@@ -1,156 +1,190 @@
+    ...
+    ```
+
+    The Kafka Topic that is created by the Channel contains the name of the namespace, `default` in this example, followed by the name of the Channel. In the consolidated Channel implementation, it is also prefixed with `knative-messaging-kafka` to indicate that it is a Kafka Channel from Knative.
+
+    !!! note
+        The topic of a Kafka Channel is an implementation detail and records from it should not be consumed from different applications.
+
+## Configuring the Knative Broker for use with Apache Kafka Channels
+
+The following example uses a ApiServerSource to publish events to the Broker, and a Trigger that routes events to a Knative Service.
+<!--TODO: Not sure this example makes sense, why would you have an event source AND channels?-->
+
+1. Create a Broker that uses Kafka Channels by default:
+
+    ```yaml
+    apiVersion: eventing.knative.dev/v1
+    kind: Broker
+    metadata:
+     name: default
+    ```
+
+1. Create a Knative Service:
+
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: broker-kafka-display
+    spec:
+      template:
+        spec:
+          containers:
+          - image: gcr.io/knative-releases/knative.dev/eventing/cmd/event_display
+    ```
+
+1. Create a ServiceAccount, ClusterRole, and ClusterRoleBinding for the ApiServerSource:
+
+    ```yaml
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: events-sa
+      namespace: default
+
+    ---
+
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: event-watcher
+    rules:
+    - apiGroups:
+      - ""
+      resources:
+      - events
+      verbs:
+      - get
+      - list
+      - watch
+
+    ---
+
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: k8s-ra-event-watcher
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: event-watcher
+    subjects:
+    - kind: ServiceAccount
+      name: events-sa
+      namespace: default
+    ```
+
+1. Create an ApiServerSource that sends events to the default Broker:
+
+    ```yaml
+    apiVersion: sources.knative.dev/v1
+    kind: ApiServerSource
+    metadata:
+      name: testevents-kafka-03
+      namespace: default
+    spec:
+      serviceAccountName: events-sa
+      mode: Resource
+      resources:
+      - apiVersion: v1
+        kind: Event
+      sink:
+        ref:
+          apiVersion: eventing.knative.dev/v1
+          kind: Broker
+          name: default
+    ```
+
+1. Create a Trigger that filters events from the Broker to the Service:
+
+    ```yaml
+    apiVersion: eventing.knative.dev/v1
+    kind: Trigger
+    metadata:
+      name: testevents-trigger
+      namespace: default
+    spec:
+      broker: default
+      subscriber:
+        ref:
+          apiVersion: serving.knative.dev/v1
+          kind: Service
+          name: broker-kafka-display
+    ```
+
```

Comment:
Should there be `kubectl apply` commands here, as stated in [Documenting YAML](https://knative.dev/docs/help/contributor/style-guide/documenting-code/#documenting-yaml) in the Style Guide?

Reply:
see https://github.com/knative/docs/pull/4028#discussion_r675155520

Reply:
Ended up just adding them since this PR has been blocked on reviews for nearly two weeks anyway 🤦🏻 🤷🏻‍♀️